### DISCUSSION TITLE
How to configure RTX + RubyMine?

### DISCUSSION THREAD

**Discussion by cmer** [jdx/mise#1289] [Q&A]
Has anyone done this successfully? I'm not sure where to start. Thanks!

**Comment by delicb** [comment]
Yep, I am using it. Nothing special to it, installed ruby in version I need, created `.mise.toml` (earlier `.rtx.toml`) selecting that version and added Ruby SDK to point to it in `~/.local/share/mise/installs/ruby/<version>/bin/ruby` (shims did not work, since RubyMine complained that it is not valid Ruby SDK, I expect it did not find surrounding folders and files it needed). 

I suspect that `.mise.toml` is probably not needed, since RubyMine is pointed directly to the installation, but I need it anyway (for environment variables management) so I created it. 

Do you have any specific problems setting it up?
